test/org/loklak/geo/GeoLocationTest.java

```diff
@@ -0,0 +1,47 @@
+package org.loklak.geo;
+
+import org.loklak.geo.GeoLocation;
+
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Collection;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * This is a test file for testing methods of org.loklak.geo.GeoLocation.java
+ */
+public class GeoLocationTest {
+
+    @Test
+    public void geoLocation() {
+        
+        Collection<String> names =  new ArrayList<String>();
+        names.add("Indian Institute of Information Technology Vadodara");
```

Comment:
Please go with something that is more static/known such as some popular historical monument. The tests are supposed to be running independent of changes in entities not a part of the project.